src/table_functions.cpp

```diff
@@ -157,15 +181,26 @@ void insert(const CallContext& ctx) {
     auto pos = ctx.arguments().get(1);
     auto value = ctx.arguments().get(2);
 
+    // Casulty because we return nil if no argument is given, but nil could be inserted. This edge
+    // case throws an error in our program, in lua the insertion works as intended I don't have an
+    // idea how to do that besides this way.
```

Comment:
You can check if `ctx.arguments().size() < 3`. Then there is no argument with index 2.

Also what is "casulty"?

Reply:
Casulty = Kollateralschaden (ist gemeint)
Something that we have to sacrifice in order to accomplish the goal.

Reply:
Then it's "casualty".

Reply:
Then it's "casualty". But doesn't really matter. The point about checking the size is more important.